scp/pubspec.yaml

```diff
@@ -15,24 +15,25 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-  camera: ^0.5.7
-  cloud_firestore: ^2.5.2
+  camera: ^0.10.0+4
```

Comment:
I see u commented on the use of camera code and if we aren't planning to use that in the future, better to remove these as well. If we're planning to use them again, comment it out for now, with a comment message.

Reply:
Alright I am removing all the unused dependencies